src/blockUser.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+var utils = require("../utils");
+var log = require("npmlog");
+
+module.exports = function(defaultFuncs, api, ctx) {
+  return function blockUser(userID, callback) {
+
+    if (!callback) {
+      callback = function() {};
+    }
+
+    defaultFuncs
+      .post("https://www.facebook.com/nfx/block_messages/?thread_fbid=" + userID + "&location=www_chat_head", ctx.jar, {})
+      .then(utils.saveCookies(ctx.jar))
+      .then(utils.parseAndCheckLogin(ctx.jar, defaultFuncs))
+      .then(function(resData) {
+        if (resData.error) {
+          throw resData;
+        }
+        defaultFuncs
+          .post("https://www.facebook.com" + /action="(.+?)"+?/.exec(resData.jsmods.markup[0][1].__html)[1].replace(/&amp;/g, "&"), ctx.jar, {})

```

Comment:
`qs` works on POST too, it's a shortcut of QueryString.
From request documentation:
`qs - object containing querystring values to be appended to the uri`